### ISSUE TITLE
`forceCreate` arg not usable when creating new Polygon from JSON `coordinates` field

### ISSUE THREAD

**Issue by brandonb927** [funiq/geoPHP#5]
It seems as though, after attempting to use this fork to parse Twitter API GeoJSON data from [this](https://developer.twitter.com/en/docs/twitter-api/enterprise/data-dictionary/native-enriched-objects/geo) page (which parses fine in the original version of geoPHP), I am unable to use it for my use case. The `coordinates` in the example below are ripped directly from the Twitter API docs.

```
$json = "{
    \"coordinates\": [
        [
            [-74.026675, 40.683935],
            [-74.026675, 40.877483],
            [-73.910408, 40.877483],
            [-73.910408, 40.3935]
        ]
    ],
    \"type\": \"Polygon\"
}";

geoPHP::load(json_decode($json, true), 'json'); // <= This throws and exception with 'Cannot create Polygon: contains non-closed ring'
```

My understanding of the new `Polygon` class is that the constructor allows passing of a param `'forceCreate' => true` to allow the Polygon to resolve itself to the start point, however there is no way to pass this arg through from `geoPHP::load`.

**Comment by mprins** [comment]
`load()` has a variable arguments list as the last argument, does that not work for you?

https://github.com/funiq/geoPHP/blob/facd8fbb0656a9c7b862870ef9fe5a4a865ae25e/src/geoPHP.php#L129-L142

**Comment by brandonb927** [comment]
> `load()` has a variable arguments list as the last argument, does that not work for you?
> 
> https://github.com/funiq/geoPHP/blob/facd8fbb0656a9c7b862870ef9fe5a4a865ae25e/src/geoPHP.php#L129-L142

Yes, using the original example data, I have tried the following

```php
geoPHP::load(json_decode($json, true), 'json', ['forceCreate' => true];
```

but it did not change anything.

**Comment by BathoryPeter** [comment]
Both OGC specification and [GeoJSON specification](https://www.rfc-editor.org/rfc/rfc7946#page-9) requires polygon rings to be closed, but I don't think Twitter has any developers left who can fix this :)

Extra arguments of `geoPHP::load()` are passed to adapters only. They can be used to control input/output format (e.g. endianness), not to modify the internal behaviour of the adapter.

I think we should add a static option to geoPHP class, e.g. `tolerantMode` and change adapters and geometries to tolerate some non critical anomalies in input data with this mode enabled.

**Comment by brandonb927** [comment]
> I think we should add a static option to geoPHP class, e.g. `tolerantMode` and change adapters and geometries to tolerate some non critical anomalies in input data with this mode enabled.

That sounds like a non-trivial amount of work that I doubt is high on anyone's priority list, so I will just keep tabs on this issue to see if anyone else runs into this like I did and if they came up with a solution.

**Comment by brandonb927** [comment]
What could be a good stop-gap in the meantime though is having the ability to configure per-type options and pass them through the `geoPHP::load()` args like 
```php
geoPHP::load(
	json_decode($json, true),
	'json',
	[
		'typeArgs' => {
			'Polygon' => [
				'forceCreate' => true,
			],
		],
	],
);
